docs/src/components/code-examples.test.fixture.js

```diff
@@ -0,0 +1,97 @@
+const mockGraphQLResponse = {
+  allCodeSamples: {
+    edges: [
+      {
+        node: {
+          content:
+            'import au.gov.notify.NotifyClient;\n\nNotifyClient client = new NotifyClient(apiKey);\n',
```

Comment:
I like that this is real content, but wonder if it's just worth keeping it more concise for the test fixtures? E.g. even the `Java.md` fixture really only needs to have content like `hi I am _markdown_` if you were going to use this fixture in the future to check that it renders the correct HTML.

Reply:
I can see your point, but I also see an advantage in keeping it close to real thing. Taking your example if we need to add more tests we're able to use this fixture without changing it and tests that are already using it don't need to be changed as a result.

Reply:
No worries. Let's leave it as you have it. If we find we're actually copy/pasting the real content, then it means we're probably wanting to test the real thing, so we'll see if we can import the real thing as a fixture.

Out of interest would it be easy to write `docs/src/components/code-examples.test.fixture.js` like:

```
const content = require('../../file.md')
{
        node: {
          content,
       }
}
```

Would that work?

Reply:
I think it would, but to clarify I didn't manually copy the contents from each `md` file to here; I ran the graphQL query and just pasted the response from there.